README.md

```diff
@@ -145,7 +152,7 @@ To install Relativity in your iOS project with [CocoaPods](http://cocoapods.org)
 
 ```
 platform :ios, '8.0'
-pod 'Relativity', '~> 0.9.0'
+pod 'Relativity', '~> 1.0.0'
```

Comment:
I'm not sure what the Podspec rules on this in documentation are, but consider `pod 'Relativity', '~> 1.0'` so people by default get the bugfix versions? Looks like the README was out of sync for 0.9 for a 5 bug fix releases, so people showing up at the repo root wouldn't get them.